src/formElementPlugins/fileUpload/src/ApplicationView.tsx

```diff
@@ -263,6 +231,12 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
       otherBody: fileData,
     })
   }
+  async function updateFileDescription(uniqueId: string, description: string | null) {
```

Comment:
can you move uplaodFile and updateFielDescription from component please

Reply:
Ohh, i thought they meant to be outside and din't actually have any extra params, sorry, I found it weird having the methods after return, thus commented